include/alpaka/core/BoostPredef.hpp

```diff
@@ -0,0 +1,99 @@
+/**
+ * \file
+ * Copyright 2017 Alexander Matthes
```

Comment:
You might want to update the authors to `Alexander Matthes, Benjamin Worpitz`.

Reply:
done